src/main/app/controller/courts/CourtTypesController.ts

```diff
@@ -10,10 +10,10 @@ import {CSRF} from '../../../modules/csrf';
 export class CourtTypesController {
 
 
-  emptyCourtCodeErrorMsg = 'Court code is required and must be numeric.';
+  emptyCourtCodeErrorMsg = 'Court code is required and must be numeric. eg(1234)';
```

Comment:
I think the example provided at the end of the string is not too clear. If we want to be sure the first character can't be start with zero, can word it like one of these below? Don't think an example add anything in this case.
- Court code is required and must be numeric and must not start with zero
- Court code is required and must be numeric and start with 1-9

Reply:
changed it to Court code is required and must be numeric and start with 1-9